### DISCUSSION TITLE
Marlin 2.x DEFAULT_AXIS_STEPS_PER_UNIT ?

### DISCUSSION THREAD

**Discussion by VenHeelun** [community/community#28926] [Discussions] [answered]
In the Marlin firmware 2.0.9 and above there are 3 lines in Configuration.h i don't understand.
In the [#define](https://www.youtube.com/hashtag/define) line are 4 values for DEFAULT_AXIS_STEPS_PER_UNIT & DEFAULT_MAX_FEEDRATE. X, Y, Z, Extruder1.
Above the line there is a comment that shows how you must use the [#define](https://www.youtube.com/hashtag/define) line.
That line does show more then 4 variables.
Here are the 3 lines :

/*                                      X, Y, Z [, I [, J [, K...]]], E0 [, E1[, E2...]]
#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 400, 500 }

/*                                      X, Y, Z [, I [, J [, K...]]], E0 [, E1[, E2...]]
#define DEFAULT_MAX_FEEDRATE          { 300, 300, 5, 25 }

/*                                      X, Y, Z [, I [, J [, K...]]], E0 [, E1[, E2...]]
#define DEFAULT_MAX_ACCELERATION      { 3000, 3000, 100, 10000 }

Question:
What does "I", "J", "K..." mean?

How does marlin know that the 4the value (500) is for Extruder1 and not for "I"?

When i translate the example then it should look like this:
#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 400, [I], [J], [K],  500 }
So the I, J, K... values are missing in the #define line.

In earlier versions the "I", "J", "K..." values where not in the comments. So this is something new.

I hope you can give me an answer.
Thank you

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/MarlinFirmware/Configurations? In that case you should open an issue *in that repository* so the developers see it. :slightly_smiling_face:

**Comment by VenHeelun** [comment]
This is the first time i ask a question on Github.
The link you give to me does say that my account is flagged.
I am trying to figure out how Github works, witch is not easy.
Is this not a bit croud for someone who only alks a question?
This is not very customer friendly i think and does make people think twice joining your comunity.
Next time please say what i shoud do, not flag me right from the start.

Thank you

**Reply by VenHeelun** [reply]
The link "contact support" in the red banner "Your account has been flagged." does link me to a page with questions and answers from people.
I can not find "Support" itself.
Support for me is the person who runs Github.
